packages/scripts/deploy-ihe-gw.sh

```diff
@@ -61,5 +62,30 @@ docker buildx build \
 
 popd
 
-echo "Restarting the IHE GW service"
-source ./packages/scripts/restart-ecs.sh
+echo "Restarting the IHE GW service $IHE_INBOUND_ECS_SERVICE"
+# Update the fargate service
+aws ecs update-service \
+  --no-cli-pager \
+  --region "$AWS_REGION" \
+  --cluster "$ECS_CLUSTER" \
+  --service "$IHE_INBOUND_ECS_SERVICE" \
+  --force-new-deployment
+
+echo "Restarting the IHE GW service $IHE_OUTBOUND_ECS_SERVICE"
+# Update the fargate service
+aws ecs update-service \
+  --no-cli-pager \
+  --region "$AWS_REGION" \
+  --cluster "$ECS_CLUSTER" \
+  --service "$IHE_OUTBOUND_ECS_SERVICE" \
+  --force-new-deployment
```

Comment:
Why do we have a distinction between inbound/outbound? If this is doing what I think it is, then limiting all our outbound bandwidth to 1 instance will be a huge bottleneck until we can scale horizontally (right now we're limited to 2 instances)

Reply:
Because we couldn't add more than 5 ports to a single service through CDK.

re: scaling
There's no relationship between the amount of services on the cluster and the amount of tasks we can run in total. Does it answer your question about it?